src/test/java/org/mamute/infra/NormalizerBrutalTest.java

```diff
@@ -27,6 +32,9 @@ public void should_remove_question_marks_and_bangs() {
 		String text = "como faz para normalizar uma string?!";
 		String slug = NormalizerBrutal.toSlug(text);
 		assertEquals("como-faz-para-normalizar-uma-string", slug);
+
+		String slugEncoded = NormalizerBrutal.toSlug(text, true);
+		assertEquals("como-faz-para-normalizar-uma-string%3f%21", slugEncoded);
```

Comment:
We can support non-latin languages by changing one line of code and adding another in NormalizerBrutal. 
[NormalizerBrutal.java.txt](https://github.com/caelum/mamute/files/95251/NormalizerBrutal.java.txt)

All tests pass except for this one